packages/harpguru-core/src/hooks/use-sizes/use-sizes.ts

```diff
@@ -1,7 +1,6 @@
 import { useGlobal } from 'reactn'
```

Comment:
I need to do a speed test across several app interactions to make sure that I haven't degraded the update speed too much. The comparison needs to be with the master branch.

Reply:
# `add-chromatic-harps-simplify-octave-groups-util` branch - `yarn start` from project root (ie dev mode)

## C major diatonic, 2nd position

- Unfragment - 2.95, 2.69, 2.69, 2.76, 2.70
- Fragment - 2.83, 2.83, 2.76, 2.78, 2.83

- 3rd position - 2.71, 2.76, 2.63, 2.77, 2.70
- 2nd position - 2.69, 2.90, 2.63, 2.57, 2.63

- 16 hole solo - 4.87, 5.06, 4.80, 4.72, 5.12
- Major diatonic - 4.60, 4.46, 4.53, 4.54, 4.40

- Country - 2.57, 2.50, 2.56, 2.90, 2.63
- Major diatonic - 2.64, 2.58, 2.44, 2.62, 2.50

- 12holes - 2.57, 2.64, 2.56, 2.84, 2.52
- fit - 2.50, 2.50, 2.50, 2.50, 2.44

- 7holes - 3.25, 3.23, 3.16, 3.10, 3.43
- fit - 3.41, 3.43, 3.44, 3.36, 3.55

- 3-10 - 3.55, 3.55, 3.49, 3.55, 3.87
- 1-7 - 3.62, 3.63, 3.61, 3.55, 3.68
 
## C 16 hole solo, 2nd positition

- Unfragment - 3.30, 3.35, 3.35, 3.29, 3.35
- Fragment - 3.35, 3.36, 3.35, 3.28, 3.43, 

- 7holes - 3.23, 3.04, 3.24, 3.16, 3.75
- fit - 3.81, 3.67, 3,75, 3.62, 3.03, 3.10

- 10-16 - 3.29, 3.24, 3.29, 3.22, 3.36, 
- 1-7 - 3.22, 3.29, 3.36, 3.29, 3.30

- 5-11 - 3.23, 3.42, 3.28, 3.29, 3.42, 
- 1-7 - 3.37, 3.30, 3.76, 3.35, 3.36

Reply:
## `v9.1.0` tag - `yarn start` from project root (ie dev mode)

### C major diatonic, 2nd position

- Unfragment - 2.76, 2.51, 2.57, 2.56, 2.84
- Fragment - 2.78, 2.64, 2.59, 2.57, 2.64

- 3rd position - 2.82, 2.77, 2.84, 2.91, 2.77 
- 2nd position - 2.77, 2.83, 3.04, 2.76, 2.69

## `v10.0.0` tag - `yarn start` from project root (ie dev mode)

### C major diatonic, 2nd position

- Unfragment - 2.77, 2.57, 2.70, 2.63, 2.70
- Fragment - 2.76, 2.76, 2.76, 2.76, 2.69

- 3rd position - 2.51, 2.50, 2.56, 2.51, 2.45
- 2nd position - 2.51, 2.65, 2.51, 2.45, 2.55

- 16 hole solo - 4.67, 4.28, 4.34, 4.33, 4.60
- Major diatonic - 4.02, 3.95, 3.88, 4.08, 3.95

- Country - 0.73, .... need I say more
- Major diatonic - 0.74,  .... "

- 12holes - NA
- fit - NA

- 7holes - NA
- fit - NA

- 4-10 - NA
- 1-7 - NA
 
### C 16 hole solo, 2nd positition

- Unfragment - 3.21, 3.22, 3.35, 3.22, 3.23
- Fragment - 3.36, 3.36, 3.22, 3.42, 3.22

- 7holes - NA
- fit - NA

- 10-16 - NA 
- 1-7 - NA

- 5-11 - NA
- 1-7 - NA

Reply:
It looks like things have generally slowed down very slightly. There is about a <10% increase in the fragmentation time and about a 10% increase in position changes on a major diatonic.

I don't really like it, but I can live with that.

There is a glaring increase in the major diatonic to country tuning time. I expect this is because we're no longer caching the harpface because it needs refreshing when we move from chromatic to diatonic. This is possibly the cause of the 10% increase in the other render times mentioned too..

This isn't necessarily a killer because the time is still in line with most of the other changes, especially the most common ones like positional changes. It would be nice to dedicate some time to getting these running faster though.

This is of course in dev mode so it's not as fast as production.

Reply:
Issue raised (https://github.com/js-jslog/harpguru/issues/135), but I'm not blocking this release. The speed decreases are acceptable in the most common usecases. We may also see an improvement following #134 .